source/loop/statsd.rst

```diff
@@ -0,0 +1,34 @@
+======
+Statsd
+======
+
+Loop-Server have got a number of Statsd counters and timers that can
+help monitor what's going on in near real-time.
+
+Statsd Counters
+###############
+
+Here is a list of the statsd counters loop-server provides:
+
+- ``loop-activated-users``: Incremented when a new Hawk Session is created for a user.
```

Comment:
I guess that could be an improvement for later. Using dotted namespaces is the standard in all statsd-based tools for filtering and searching the produced data